server/util/grpc_server/grpc_server.go

```diff
@@ -146,6 +147,8 @@ func NewGRPCServer(env environment.Env, port int, credentialOption grpc.ServerOp
 
 	regServices(grpcServer, env)
 
+	channelzservice.RegisterChannelzServiceToServer(grpcServer)
```

Comment:
does this get all grpc servers, or just one of internal/external?

Reply:
woops, this does register the channelz service to all servers but I wasn't configuring the HTTP handler correctly to actually expose the channelz page for each server. Fixed.